src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/CompareType.java

```diff
@@ -102,10 +104,11 @@ private CompareType(CompareUtil util) {
      * Tells if the given string matches the given pattern based on the algorithm of this CompareType instance.
      * @param pattern the pattern
      * @param str the string
+     * @param envVars the environment variables exisiting on the jenkins host.
      * @return true if the string matches the pattern.
      */
-    public boolean matches(String pattern, String str) {
```

Comment:
The old signature needs to be kept and deprecated for backwards binary compatibility